src/install/lockfile.zig

```diff
@@ -1704,8 +1704,11 @@ pub const FormatVersion = enum(u32) {
     // bun v0.1.7+
     // This change added tarball URLs to npm-resolved packages
     v2,
+    // bun v0.6.8+
+    // Store life cycle scripts of resolved packages
+    v3,
     _,
-    pub const current = FormatVersion.v2;
+    pub const current = FormatVersion.v3;
```

Comment:
Couldn't we do an in-place migration here to add the extra field and then a marker at the very end of the file that denotes whether this field will exist? As-is, this will automatically invalidate all existing lockfiles in Bun which shouldn't strictly be necessary

Reply:
except those older lockfiles would cause dependency life cycle scripts to not execute, so keeping them around will keep this bug alive in the wild long after this fix